### ISSUE TITLE
Documentation Accessibility Pass

### ISSUE THREAD

**Issue by cole-nelson** [Purdue-SoCET/RISCVBusiness#149]
We need to clean up any documentation for better accessibility. As most documentation is in Markdown, most issues will come down to missing elements like table headers, link text, image alt-text. We should also ensure that diagrams have reasonable contrast.

The following link is useful: https://www.w3.org/WAI/tutorials/images/decision-tree/

Main tasks:
- [ ] Fix existing documentation
- [ ] Add lint check for documentation to address obvious issues
- [ ] Look in to mdbook theming for better default

**Comment by wrcunnin** [comment]
Pre-existing documentation locations (mark when done)
- [ ] https://github.com/Purdue-SoCET/RISCVBusiness/tree/master/docs/src
- [ ] https://github.com/Purdue-SoCET/RISCVBusiness/tree/master/docs/src/extensions
- [ ] https://github.com/Purdue-SoCET/RISCVBusiness/tree/master/docs/src/hpms
- [ ] https://github.com/Purdue-SoCET/RISCVBusiness/tree/master/docs/src/prediction
- [ ] https://github.com/Purdue-SoCET/RISCVBusiness/tree/master/docs/src/rv32e
- [ ] https://github.com/Purdue-SoCET/RISCVBusiness/tree/master/docs/src/supervisor
- [ ] https://github.com/Purdue-SoCET/RISCVBusiness/blob/master/README.md
- [ ] https://github.com/Purdue-SoCET/RISCVBusiness/blob/master/source_code/README.md
